pubsub/testing.go

```diff
@@ -1,16 +1,32 @@
-package ps
+package pubsub
```

Comment:
You may want to rename this file to `pubsub/subscriber_test.go` to match our naming convention

Reply:
This is not the tests, this is test support code.